### DISCUSSION TITLE
Obsidian-style editing

### DISCUSSION THREAD

**Discussion by pburrows** [mdx-editor/editor#169] [General]
Is there a way to enable an "Obsidian-style" editing mode where you can still type and edit Markdown, but see the correctly rendered markdown in the editor?

**Comment by petyosi** [comment]
No, you're probably looking for a different kind of a solution here. This is pure wysiwyg. Notice [that some markdown shortcuts work, though](https://mdxeditor.dev/editor/docs/markdown-shortcuts).
